tests/component/test_export_signal_properties.py

```diff
@@ -0,0 +1,81 @@
+"""Contains a collection of pytest tests that validates export signal properties."""
+
+import pytest
+
+import nidaqmx
+from nidaqmx.constants import ExportAction
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+from nidaqmx.system.device import Device
+
+
+def test__export_signal__get_enum_property__returns_value(any_x_series_device: Device):
+    """Test to validate getter for export signal property of enum type."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+        assert task.export_signals.samp_clk_output_behavior == ExportAction.PULSE
+
+
+def test__export_signal__set_enum_property__returns_assigned_value(any_x_series_device: Device):
+    """Test to validate setter for export signal property of enum type."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+        value_to_set = ExportAction.LEVEL
+        task.export_signals.samp_clk_output_behavior = value_to_set
+
+        assert task.export_signals.samp_clk_output_behavior == value_to_set
+
+
+def test__export_signal__reset_enum_property__returns_initial_value(any_x_series_device: Device):
+    """Test to validate resetting export signal property of enum type."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+        del task.export_signals.samp_clk_output_behavior
```

Comment:
this test currently could be passing if reset is a noop - set to non-default (you don't need to validate that worked since another test does that) before reset

Reply:
Resolved. Set non-default value before resetting.